layouts/shortcodes/blocks/feature.html

```diff
@@ -2,10 +2,12 @@
 {{ $url_text := .Get "url_text" -}}
 <div class="col-lg-4 mb-5 mb-lg-0 text-center">
 <div class="mb-4 h1">
+<a class="text-white" target="_blank" href="{{ .Get "url" }}">
```

Comment:
Currently, the link is always applied, even if shortocde parameter `url` is not given.
Please use `with` function here to apply link only if parameter `url` is given.

Reply:
As asked, I had made the changes, please review it.